components/popover/index.css

```diff
@@ -14,94 +14,106 @@ governing permissions and limitations under the License.
 }
 
 /* windows high contrast mode */
 @media (forced-colors: active) {
-  .spectrum-Popover {
-    --highcontrast-popover-border-color: CanvasText;
-  }
+	.spectrum-Popover {
+		--highcontrast-popover-border-color: CanvasText;
+	}
 }
 
 .spectrum-Popover {
-  --spectrum-popover-filter: drop-shadow(var(--mod-popover-shadow-horizontal, var(--spectrum-popover-shadow-horizontal)) var(--mod-popover-shadow-vertical, var(--spectrum-popover-shadow-vertical)) var(--mod-popover-shadow-blur, var(--spectrum-popover-shadow-blur)) var(--mod-popover-shadow-color, var(--spectrum-popover-shadow-color)));
-  @extend %spectrum-overlay;
-
-  box-sizing: border-box;
-  position: absolute;
-
-  outline: none; /* Hide focus outline */
-
-  padding: var(--mod-popover-content-area-spacing-vertical, var(--spectrum-popover-content-area-spacing-vertical)) 0;
-
-  display: inline-flex;
-  flex-direction: column;
-
-  border-style: solid;
-  border-radius: var(--mod-popover-corner-radius, var(--spectrum-popover-corner-radius));
-  border-color: var(--highcontrast-popover-border-color, var(--mod-popover-border-color, var(--spectrum-popover-border-color)));
-  border-width: var(--mod-popover-border-width, var(--spectrum-popover-border-width));
-
-  background-color: var(--mod-popover-background-color, var(--spectrum-popover-background-color));
-  filter: var(--mod-popover-filter, var(--spectrum-popover-filter));
-
-  /* has tip triangle */
-  &.spectrum-Popover--withTip {
-    .spectrum-Popover-tip {
-      /* triangle polygon */
-      .spectrum-Popover-tip-triangle {
-        stroke-linecap: square;
-        stroke-linejoin: miter;
-        fill: var(--highcontrast-popover-background-color, var(--mod-popover-background-color, var(--spectrum-popover-background-color)));
-        stroke: var(--highcontrast-popover-border-color, var(--mod-popover-border-color, var(--spectrum-popover-border-color)));
-        stroke-width: var(--mod-popover-border-width, var(--spectrum-popover-border-width));
-      }
-    }
-  }
+	--spectrum-popover-filter: drop-shadow(var(--mod-popover-drop-shadow-x, var(--spectrum-popover-drop-shadow-x)) var(--mod-popover-drop-shadow-y, var(--spectrum-popover-drop-shadow-y)) var(--mod-popover-drop-shadow-blur, var(--spectrum-popover-drop-shadow-blur)) var(--mod-popover-drop-shadow-color, var(--spectrum-popover-drop-shadow-color)));
+	@extend %spectrum-overlay;
+
+	box-sizing: border-box;
+	position: absolute;
+
+	outline: none; /* Hide focus outline */
+
+	padding: var(--mod-popover-content-area-spacing, var(--spectrum-popover-content-area-spacing));
+
+	display: inline-flex;
+	flex-direction: column;
+
+	border-style: solid;
+	border-radius: var(--mod-popover-corner-radius, var(--spectrum-popover-corner-radius));
+	border-color: var(--highcontrast-popover-border-color, var(--mod-popover-border-color, var(--spectrum-popover-border-color-with-transparency)));
+
+	/* border-color: var(--spectrum-popover-border-color-default); */
```

Comment:
A comment here about why this is commented out would be nice, but feel free not to add one if you disagree.

Reply:
It was an extraneous comment from earlier edits, trying to popover-border-color-default, which is one of those rgba() tokens.

Reply:
Oh! I thought it went with this one, and if we were able to get to a point where we could use that, we'd be able to use the border-color that was commented out:
```
/* TODO: Cannot use popover-border-color token until the value no longer has the rgba() function alpha/opacity */
/* --spectrum-popover-border-color-default: var(--spectrum-popover-border-color); */
```

Reply:
Yes, it was related to that `TODO` comment. `popover-border-color` doesn't have the transparency built in though, which is why I went with the rgb token instead. I added a little clarity in that comment- does that make more sense? https://github.com/adobe/spectrum-css/pull/3365/commits/05fc517e22ee3c0b93ebcb41f3d3e931a37d808c